sidebars/core-concepts.js

```diff
@@ -71,6 +71,7 @@ export const coreConcepts = [
   "developer/taxes",
   "developer/thumbnails",
   "developer/price-calculation",
+  "developer/price-expiration",
```

Comment:
It regards just orders, right? So maybe it should go to `Order` section instead

Reply:
It will work in the same way? If not, I think it's better to create new page for checkout and put it in the checkout section